tests/integration/list/list_integ_base.py

```diff
@@ -0,0 +1,34 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+
+
+class ListIntegBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cmd = cls.base_command()
+        cls.test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "list")
+
+    def setUp(self):
+        super().setUp()
+        self.scratch_dir = str(Path(__file__).resolve().parent.joinpath(str(uuid.uuid4()).replace("-", "")[:10]))
+        shutil.rmtree(self.scratch_dir, ignore_errors=True)
+        os.mkdir(self.scratch_dir)
+
+        self.working_dir = tempfile.mkdtemp(dir=self.scratch_dir)
+
+    def tearDown(self):
+        super().tearDown()
+        self.working_dir and shutil.rmtree(self.working_dir, ignore_errors=True)
+        self.scratch_dir and shutil.rmtree(self.scratch_dir, ignore_errors=True)
+
+    @classmethod
+    def base_command(cls):
+        command = "sam"
+        if os.getenv("SAM_CLI_DEV"):
+            command = "samdev"
+
+        return command
```

Comment:
Let's use `get_sam_command` from `testing_utils.py` file.

Reply:
Changed it to get_sam_command